ambry-clustermap/src/main/java/com/github/ambry/clustermap/HelixClusterManagerMetrics.java

```diff
@@ -190,15 +196,104 @@ void initializePartitionMetrics() {
           }
         }
         if (downReplicaCount > replicaCount / 2) {
-            logger.info("There are more than half of the replicas are down for partition {}, the down replicas are {}",
-                partition.toPathString(), downReplicas);
+          logger.info("There are more than half of the replicas are down for partition {}, the down replicas are {}",
+              partition.toPathString(), downReplicas);
           return 1L;
         }
       }
       return 0L;
     };
     registry.gauge(MetricRegistry.name(HelixClusterManager.class, "isMajorityReplicasDownForAnyPartition"),
         () -> isMajorityReplicasDownForAnyPartition);
+
+    Gauge<Integer> getLocalPartitionCountWithOneReplicaDown = () -> {
+      String dcName = clusterMapCallback.getDatacenterName();
```

Comment:
you wanna clear the map every time this method is called. so the previous value won't affect each invocation.